### PR TITLE
Replaces home dir with tilde shorthand

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -58,4 +64,4 @@ fn it_works() {
     assert_eq!(tico("~/work/personal/tico/"), "~/w/p/t/");
     assert_eq!(tico("~/work/ééé/tico"), "~/w/é/tico");
     assert_eq!(tico("~/.config/htop"), "~/.c/htop");
-}
```
Comment (hugopeixoto) [inline src/lib.rs]: :thinking:

File: src/lib.rs [hunk 2]
```diff
@@ -10,7 +12,11 @@
 /// println!("{}", shortened);
 /// // => "/h/.s/path"
 /// ```
-pub fn tico(tico: &str) -> String {
+pub fn tico(path: &str) -> String {
+    let home_dir = dirs::home_dir().unwrap();
+    let home_dir_str = home_dir.to_str().unwrap();
```
Comment (hugopeixoto) [inline src/lib.rs]: Do we need both `unwrap`s? Can `#to_str` be called on `dirs::home_dir()` directly?

Comment (locks) [reply src/lib.rs]: It cannot, `dirs::home_dir()` returns an `Option`. I am sure this can be improved tho.

File: src/lib.rs (hunk 1 above)
Comment (locks) [reply src/lib.rs]: Was fixed but not pushed.

**Comment by hugopeixoto** [comment]
:tada:

**Pull request merged**